arkane/common.py

```diff
@@ -32,33 +28,36 @@
 #                                                                             #
 ###############################################################################
 
-import numpy
-import os.path
+"""
+Arkane common module
+"""
+
 import logging
-import time
+import os.path
 import string
-
+import time
 import yaml
 
-from rmgpy.rmgobject import RMGObject
+import numpy as np
+
+import rmgpy.constants as constants
 from rmgpy import __version__
-from rmgpy.quantity import ScalarQuantity, ArrayQuantity
 from rmgpy.molecule.element import elementList
 from rmgpy.molecule.translator import toInChI, toInChIKey
+from rmgpy.pdep.collision import SingleExponentialDown
+from rmgpy.quantity import ScalarQuantity, ArrayQuantity
+from rmgpy.rmgobject import RMGObject
+from rmgpy.species import Species, TransitionState
 from rmgpy.statmech.conformer import Conformer
 from rmgpy.statmech.rotation import LinearRotor, NonlinearRotor, KRotor, SphericalTopRotor
 from rmgpy.statmech.torsion import HinderedRotor, FreeRotor
 from rmgpy.statmech.translation import IdealGasTranslation
 from rmgpy.statmech.vibration import HarmonicOscillator
-from rmgpy.pdep.collision import SingleExponentialDown
-from rmgpy.transport import TransportData
 from rmgpy.thermo import NASA, Wilhoit, ThermoData, NASAPolynomial
-from rmgpy.species import Species, TransitionState
-import rmgpy.constants as constants
+from rmgpy.transport import TransportData
 
 from arkane.pdep import PressureDependenceJob
```

Comment:
I think we should separate them just because we'd like to think of Arkane and RMG as different software, although I agree with PyCharm that technically it's the same. I consciously chose to separate them, but I'm open to different opinions

Reply:
I am fine with them being a separate block then